content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Add a new Custom Ingress Gateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provides instructions to manually create a new custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
+This task was validated on AWS Infrastructure.
```

Comment:
I think if the task is not specific for AWS, it should be tested on some "generic" Kubernetes cluster, without cloud provider specific annotations. In needed, as a remark, the instructions for AWS could be provided, after the instructions for a generic Kubernetes cluster.